badparking/mobile_api/views.py

```diff
@@ -1,3 +1,40 @@
-from django.shortcuts import render
+from django.http import Http404
+from django.contrib.auth.models import User
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework import status
 
-# Create your views here.
+from core.models import Claim, CrimeType
+from media.models import MediaFileModel
+from .serializers import ClaimSerializer
+
+
+class ClaimList(APIView):
+    def get(self, request, format=None):
+        claims = Claim.objects.all()
+        serializer = ClaimSerializer(claims, many=True)
+        return Response(serializer.data)
+
+    def post(self, request, format=None):
+        serializer = ClaimSerializer(data=request.data)
+
+        if serializer.is_valid():
+            claim = Claim(**serializer.data)
```

Comment:
Please read up the [docs](http://www.django-rest-framework.org/api-guide/serializers/) on REST framework serializers. It's possible for a serializer to have customised saving routines with additional logic. Moreover, `serializer.data` is basically just `request.data` in this case... if saving manually you should've been using `serializer.validated_data`.